src/tree/AttachedAccountsTreeItem.ts

```diff
@@ -157,7 +150,7 @@ export class AttachedAccountsTreeItem extends AzureParentTreeItem {
                 prompt: 'Enter the connection string for your database account',
                 validateInput: validateInput,
                 ignoreFocusOut: true,
-                value: defaultValue
+                value: placeholder
```

Comment:
I feel like this should be either `mongodb://127.0.0.1:27017` or undefined, not the `placeHolder`